app/models/dashboard.rb

```diff
@@ -2,6 +2,14 @@
 #  Class to handle dashboard metrics
 ###
 class Dashboard
+  METRIC_TYPE = {
+    custom: 'Custom',
+    hold_recalls: 'HoldRecall',
+    mediated_pages: 'MediatedPage',
+    pages: 'Page',
+    scans: 'Scan'
+  }.freeze
```

Comment:
This seems a bit strange to have.   Can we just use `classify` on the string representation of these metrics?

```
:custom.to_s.classify
=> "Custom"

:hold_recalls.to_s.classify
=> "HoldRecall"
```